tensorflow_hub/tools/make_image_classifier/make_image_classifier.py

```diff
@@ -119,7 +124,8 @@ def _get_hparams_from_flags():
       batch_size=FLAGS.batch_size,
       learning_rate=FLAGS.learning_rate,
       momentum=FLAGS.momentum,
-      dropout_rate=FLAGS.dropout_rate)
+      dropout_rate=FLAGS.dropout_rate,
+      set_memory_growth=FLAGS.set_memory_growth)
```

Comment:
This should fail now, after the extension to HParams has been removed.

Please fix and be sure to run the changed code for testing.